src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -216,6 +216,16 @@ public void DisplayMessage(string messageId)
             MessageCenterClass.Shared().ShowMessageCenter(messageId);
         }
 
+        public void MarkMessageRead(string messageId)
+        {
+            MessageCenterClass.Shared().Inbox.GetMessage(messageId).MarkRead();
```

Comment:
I think you will face NPE if the message is not available. Try:
`MessageCenterClass.Shared().Inbox.MarkMessagesRead/DeleteMessages`